Methods/Parameters/Inventory/Inventory/Program.cs

```diff
@@ -0,0 +1,314 @@
+
+
+
+//int choice = 0;
+//char choice2 = 'x';
+//int updatechoice = 0;
+//int updatedvalue = 0;
+//string item = null;
+//int Quantity = 0;
+//int Price = 0;
+
+while (gameon) {
+
+    Console.Clear();
+    MainMenu();
+    while (!int.TryParse(Console.ReadLine(), out choice))
+    {
+        Console.WriteLine("Invalid input. Please enter a valid number (1-7)");
+    }
+    Console.Clear();
+
+switch (choice)
+{
+    case 1:
+        ViewInventory();
+        break;
+    case 2:
+           AddItemDisplay();
+            //var (name, quantity, price) = AddItemDisplay();
+            //Additem(name, quantity, price);
+            break;
+    case 3:
+            RemoveItem();
+        break;
+    case 4:
+        UpdateItem();
+        break;
+    case 5:
+        SearchItem();
+        break;
+    case 6:
+        ViewTotal();
+        break;
+    case 7:
+            Console.Clear();
+            Console.WriteLine("Goodbye.");
+        gameon = false;
+        break;
+    default:
+        break;
+}
+}
+
+void CalculateTotal()
+{
+    totalvalue = 0;
+
+    foreach (Item i in items)
+    {
+        
+        n = i.Quantity * i.Price;
+        totalvalue += n;
+
+    }
+}
+void ViewTotal()
+{
+
+    CalculateTotal();
+    
+    Console.WriteLine(@$"
+---------------------------- Total Inventory Value ----------------------------
+Total Inventory Value: ${totalvalue}
+---------------------------------------------------------------------------");
+    ReturnToMenu();
+}
+void SearchItem()
+{
+    Console.WriteLine(@"
+    ----------------------------Search Item----------------------------
+Please enter the name of the item you want to search for:
+");
+
+    var foundItem = PrintItem();
+    Console.WriteLine(@"
+    --------------------------------------------------------------------");
+    ReturnToMenu();
+
+}
+void UpdateItem()
+{
+    Console.WriteLine(@"
+    ----------------------------Update Item----------------------------
+Please enter the name of the item you want to update:");
+    var foundItem = PrintItem();
+
+Console.WriteLine(@"
+
+What would you like to update?
+1. Quantity
+2. Price
+Please select an option (1-2):");
+
+
+    int option = 0;
+    while (!int.TryParse(Console.ReadLine(), out option) || option!=1 && option!=2)
+    {
+        Console.WriteLine("Invalid input. Please enter a valid number for quantity:");
+    }
+    
+    Console.WriteLine(@"
+Please enter the new value:");
+
+
+
+
+    int value = 0;
+    while (!int.TryParse(Console.ReadLine(), out value) || value > 100)
+    {
+        Console.WriteLine("Invalid input. Please enter a valid number for quantity:");
+    }
+    Console.WriteLine(@"
+--------------------------------------------------------------------");
+
+    if(option == 1)
+    {
+        foundItem.Quantity = value;
+    }
+    else
+    {
+        foundItem.Price = value;
+    }
+
+
+
+    Console.WriteLine($@"Item successfully updated.");
+
+    ReturnToMenu();
+
+    
+}
+void RemoveItem()
```

Comment:
The while loop for confirmation (Y/N) could be simplified and extracted into a helper method.
Ex:
`
bool GetConfirmation(string message)
{
    Console.WriteLine(message);
    string response = Console.ReadLine()?.ToUpper();
    while (response != "Y" && response != "N")
    {
        Console.WriteLine("Invalid input. Please enter Y/N:");
        response = Console.ReadLine()?.ToUpper();
    }
    return response == "Y";
}

`